lib/openstudio-standards/service_water_heating/create_piping_losses.rb

```diff
@@ -0,0 +1,155 @@
+
+      # Construction for pipe
+      pipe_construction = OpenStudio::Model::Construction.new(model)
+
+      # Add insulation material to insulated pipe
+      if pipe_insulation_thickness > 0
+        # Material for fiberglass insulation
+        # R-value from Owens-Corning 1/2in fiberglass pipe insulation
+        # https://www.grainger.com/product/OWENS-CORNING-1-2-Thick-40PP22
+        # but modified until simulated heat loss = 17.7 Btu/hr/ft of pipe with 140F water and 70F air
+        pipe_insulation_thickness_in = OpenStudio.convert(pipe_insulation_thickness, 'm', 'in').get
+        insulation = OpenStudio::Model::StandardOpaqueMaterial.new(model)
+        insulation.setName("Fiberglass batt #{pipe_insulation_thickness_in.round(2)}in")
+        insulation.setRoughness('Smooth')
+        insulation.setThickness(OpenStudio.convert(pipe_insulation_thickness_in, 'in', 'm').get)
+        insulation.setThermalConductivity(OpenStudio.convert(0.46, 'Btu*in/hr*ft^2*R', 'W/m*K').get)
+        insulation.setDensity(OpenStudio.convert(0.7, 'lb/ft^3', 'kg/m^3').get)
+        insulation.setSpecificHeat(OpenStudio.convert(0.2, 'Btu/lb*R', 'J/kg*K').get)
+        insulation.setThermalAbsorptance(0.9) # Irrelevant for Pipe:Indoor; no radiation model is used
+        insulation.setSolarAbsorptance(0.7) # Irrelevant for Pipe:Indoor; no radiation model is used
+        insulation.setVisibleAbsorptance(0.7) # Irrelevant for Pipe:Indoor; no radiation model is used
+
+        pipe_construction.setName("Copper pipe 0.75in type L with #{pipe_insulation_thickness_in.round(2)}in fiberglass batt")
+        pipe_construction.setLayers([insulation, copper_pipe])
+      else
+        pipe_construction.setName('Uninsulated copper pipe 0.75in type L')
+        pipe_construction.setLayers([copper_pipe])
+      end
+
+      heat_loss_pipe = OpenStudio::Model::PipeIndoor.new(model)
+      heat_loss_pipe.setName("#{service_water_loop.name} Pipe #{pipe_length_ft.round}ft")
+      heat_loss_pipe.setEnvironmentType('Schedule')
+      # @todoschedule type registry error for this setter
+      # heat_loss_pipe.setAmbientTemperatureSchedule(swh_piping_air_temp_sch)
+      heat_loss_pipe.setPointer(7, swh_piping_air_temp_sch.handle)
+      # @todo schedule type registry error for this setter
+      # heat_loss_pipe.setAmbientAirVelocitySchedule(model.alwaysOffDiscreteSchedule)
+      heat_loss_pipe.setPointer(8, swh_piping_air_velocity_sch.handle)
+      heat_loss_pipe.setConstruction(pipe_construction)
+      heat_loss_pipe.setPipeInsideDiameter(OpenStudio.convert(0.785, 'in', 'm').get)
+      heat_loss_pipe.setPipeLength(OpenStudio.convert(pipe_length_ft, 'ft', 'm').get)
+
+      heat_loss_pipe.addToNode(service_water_loop.demandInletNode)
+
+      OpenStudio.logFree(OpenStudio::Info, 'openstudio.model.Model', "Added #{pipe_length_ft.round}ft of #{pipe_construction.name} losing heat to #{air_temperature_f.round}F air to #{service_water_loop.name}.")
```

Comment:
Update log messages with the right module name

Reply:
changed logging messages to 'openstudio.standards.ServiceWaterHeating'